hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/pipes/TestPipeApplication.java

```diff
@@ -74,6 +75,11 @@
 
   private static String taskName = "attempt_001_02_r03_04_05";
 
+  @After
+  public void after() throws Exception {
+    cleanTokenPasswordFile();
+  }
+
```

Comment:
Big +1 for this change, but I think this issue can be fixed in a separate jira.

Reply:
Pushed the latest changes without this.
Please check